app/src/main/java/de/westnordost/streetcomplete/data/meta/ResurveyUtils.kt

```diff
@@ -13,6 +13,14 @@ fun getLastCheckDateKeys(key: String): Sequence<String> = sequenceOf(
     "$key:last_checked", "last_checked:$key"
 )
 
+val LAST_CHECK_DATE_KEYS = listOf(
+    "check_date",
+    "lastcheck",
+    "last_checked",
+    "survey:date",
+    "survey_date"
+)
```

Comment:
I put this next to getLastCheckDateKeys to ensure things would be added to both together in future.

However I see there are a handful of instances of the missing ones:
https://taginfo.openstreetmap.org/search?q=survey_date
https://taginfo.openstreetmap.org/search?q=survey:date

Would it therefore be worth me changing getLastCheckDateKeys so it does $key:i and i:$key for each i in LAST_CHECK_DATE_KEYS and avoid the duplicate lists entirely?

Reply:
Nah, it's fine

Reply:
I assume you mean not changing the function *and* not deleting those other check date sub-keys?

Reply:
Yes, not changing the function and not deleting those other check date sub keys, i.e. change nothing here.

- it makes reading the code a little more complicated
- but more importantly, it mixes tagging patterns for check dates for the whole object and namespaced tagging patterns for check-dates of single keys. OpenStreetMap tagging is not that "regular". `surveydate` is used (a lot) for the whole object, but not really used at all with namespace. We shouldn't put it in the code if it practically does not exist just to save a few lines of code / because it looks more regular that way.